workers/utils.py

```diff
@@ -319,3 +320,31 @@ def setup_environ() -> None:
     # Apply
     os.environb.clear()
     os.environb.update(env)
+
+FUZZER_CMD_PORT = 7055
+
+class _PausedFuzzers:
```

Comment:
Could you add a docstring describing what the class is doing? And how to use it?